spec/jobs/upvs/fetch_services_with_forms_list_job_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Upvs::FetchServicesWithFormsListJob, type: :job do
+  describe '#perform' do
+    let(:url) { 'https://example.com' }
+
+    let(:downloader) { double }
+
+    it 'downloads and imports ServicesWithForms list in V1 format' do
+      expect(downloader).to receive(:download_file).with(url).and_return(fixture_filepath('upvs/services-v1.csv'))
+
+      subject.perform(url, downloader: downloader)
+
+      expect(Upvs::ServiceWithForm.first).to have_attributes(
+        instance_id: 2107,
+        external_code: 'App.GeneralAgenda',
+        meta_is_code: 'NULL',
```

Comment:
toto urcite nechceme, mapujme to na `nil` (podobne nizsie)

```suggestion
        meta_is_code: nil,
```

Reply:
@smith558 inak dajme tu aj dalsi priklad kde `meta_is_code`, ktory nie je `nil` (podobne pre `info_url`)